premarum-backend/PreEnrollmentMgmt.Core/Entities/Output/PreEnrollmentCourseParserOutput.cs

```diff
@@ -0,0 +1,13 @@
+namespace PreEnrollmentMgmt.Core.Entities.Output;
```

Comment:
Maybe Rename to PreEnrollmentNonCompliantCourses ? or something similar. After rename we can move with the other computed entities in the computed entities folder